Description: Disable printing support to remove the dependency on the
 deprecated libgnomeprint library.
Author: Richard Laager <rlaager@wiktel.com>
Forwarded: no
Last-Update: 2012-03-18

Index: gbonds/src/Makefile.am
===================================================================
--- gbonds.orig/src/Makefile.am	2012-03-18 20:37:17.000000000 +0000
+++ gbonds/src/Makefile.am	2012-03-18 20:50:47.425806617 +0000
@@ -36,8 +36,6 @@ gbonds_SOURCES = \
 	recent.h			\
 	file.c				\
 	file.h				\
-	print-dialog.c			\
-	print-dialog.h			\
 	prefs-dialog.c			\
 	prefs-dialog.h			\
 	view.c				\
@@ -59,8 +57,6 @@ gbonds_SOURCES = \
 	rules.h				\
 	types.c				\
 	types.h				\
-	print.c				\
-	print.h				\
 	prefs.c				\
 	prefs.h				\
 	update.c			\
Index: gbonds/src/ui.c
===================================================================
--- gbonds.orig/src/ui.c	2012-03-18 20:37:17.000000000 +0000
+++ gbonds/src/ui.c	2012-03-18 20:51:19.376482060 +0000
@@ -53,7 +53,9 @@ static BonoboUIVerb gb_ui_verbs [] = {
 	BONOBO_UI_VERB ("FileImport",            gb_cmd_file_import),
 	BONOBO_UI_VERB ("FileSave",              gb_cmd_file_save),
 	BONOBO_UI_VERB ("FileSaveAs",            gb_cmd_file_save_as),
+#if 0
 	BONOBO_UI_VERB ("FilePrint",             gb_cmd_file_print),
+#endif
 	BONOBO_UI_VERB ("FileClose",             gb_cmd_file_close),
 	BONOBO_UI_VERB ("FileExit",              gb_cmd_file_exit),
 	BONOBO_UI_VERB ("EditCut",               gb_cmd_edit_cut),
Index: gbonds/src/commands.h
===================================================================
--- gbonds.orig/src/commands.h	2012-03-18 20:37:17.000000000 +0000
+++ gbonds/src/commands.h	2012-03-18 20:50:47.425806617 +0000
@@ -47,9 +47,11 @@ void gb_cmd_file_save_as         (Bonobo
 				  gpointer           user_data,
 				  const gchar       *verbname);
 
+#if 0
 void gb_cmd_file_print           (BonoboUIComponent *uic,
 				  gpointer           user_data,
 				  const gchar       *verbname);
+#endif
 
 void gb_cmd_file_print_preview   (BonoboUIComponent *uic,
 				  gpointer           user_data,
Index: gbonds/src/commands.c
===================================================================
--- gbonds.orig/src/commands.c	2012-03-18 20:37:17.000000000 +0000
+++ gbonds/src/commands.c	2012-03-18 20:50:47.425806617 +0000
@@ -124,6 +124,7 @@ gb_cmd_file_save_as (BonoboUIComponent *
 	gb_file_save_as (GB_VIEW(window->view)->doc, GTK_WINDOW(window));
 }
 
+#if 0
 /****************************************************************************/
 /* File->Print command.                                                     */
 /****************************************************************************/
@@ -141,6 +142,7 @@ gb_cmd_file_print (BonoboUIComponent *ui
 	gb_print_dialog (GB_VIEW(window->view), BONOBO_WINDOW(window));
 
 }
+#endif
 
 /****************************************************************************/
 /* File->Close command.                                                     */
Index: gbonds/configure.in
===================================================================
--- gbonds.orig/configure.in	2012-03-18 20:37:17.000000000 +0000
+++ gbonds/configure.in	2012-03-18 20:50:47.425806617 +0000
@@ -38,8 +38,6 @@ PKG_CHECK_MODULES(GBONDS, gtk+-2.0 >= $G
 libgnomeui-2.0 >= $LIBGNOMEUI_REQUIRED \
 libbonobo-2.0 >= $LIBBONOBO_REQUIRED \
 libxml-2.0 >= $LIBXML_REQUIRED \
-libgnomeprint-2.2 >= $LIBGNOMEPRINT_REQUIRED \
-libgnomeprintui-2.2 >= $LIBGNOMEPRINTUI_REQUIRED \
 gnome-vfs-2.0 >= $GNOMEVFS_REQUIRED \
 )
 
Index: gbonds/src/gbonds-ui.xml
===================================================================
--- gbonds.orig/src/gbonds-ui.xml	2012-03-18 17:06:18.169893838 -0500
+++ gbonds/src/gbonds-ui.xml	2012-03-18 17:07:02.350964202 -0500
@@ -19,10 +19,6 @@
 	pixtype="stock" pixname="gtk-save-as"
 	accel="*Shift**Control*S"/>
 
-        <cmd name="FilePrint" _label="Print"
-	_tip="Print the current file" pixtype="stock" pixname="gtk-print"
-	accel="*Control*P"/>
-	
         <cmd name="FileClose" _label="Close"
 	_tip="Close the current file" pixtype="stock" pixname="gtk-close"
 	accel="*Control*W"/>
@@ -114,10 +110,6 @@
 	
         <menuitem name="FileSaveAs" verb="" _label="Save _As..."/>
 
-	<separator/>
-			
-	<menuitem name="FilePrint" verb="" _label="_Print..."/>
-
 	<placeholder name="FileOps_1" />
 
 	<placeholder name="FileOps_2" delimit="top"/>
@@ -222,7 +214,6 @@
 	<separator/>
 
 	<toolitem name="FileSave" verb=""/>
-	<toolitem name="FilePrint" verb=""/>
 	
 	<separator/>
 
